src/prefect/cli/deployment.py

```diff
@@ -474,21 +488,17 @@ async def run(
     else:
         run_url = "<no dashboard available>"
 
-    if start_in:
+    try:
         scheduled_display = (
             scheduled_start_time.in_tz(
                 pendulum.tz.local_timezone()
             ).to_datetime_string()
-            + " ("
-            + pendulum.format_diff(scheduled_start_time.diff(now))
-            + ")"
+            + human_dt_diff
+        )
+    except Exception as exc:
+        exit_with_error(
+            f"Unable to format scheduled start time display from '{scheduled_start_time}': {exc!r}"
         )
```

Comment:
We probably shouldn't have a try/except here, we should always be able to format a date time object correctly. Is there a case where this can happen due to weird user input?

Reply:
I'm not sure I've exhausted the full variety of possible user input through testing/to what extent pendulum can handle all of the reformatting.